scripts/ci/docker-compose/integration-otel.yml

```diff
@@ -54,14 +54,31 @@ services:
       - ./grafana/volume/dashboards:/grafana/dashboards
       - ./grafana/volume/provisioning:/grafana/provisioning
 
+  jaeger:
+    image: jaegertracing/all-in-one
+    container_name: "breeze-jaeger"
+    environment:
+      COLLECTOR_OTLP_ENABLED: true
+      COLLLECTOR_ZIPKIN_HOST_PORT: :9411
```

Comment:
Just want to confirm that the colon is intended to be there?  It looks suspicious.

Reply:
The colon does not seem to be necessary - in 'some' documentation it was there, but I am now certain it is a typo. [Jaeger Doc](https://www.jaegertracing.io/docs/1.6/getting-started/)

Reply:
Cool.  If it works, that's fine.  It looks odd so I wanted to verify.